exchangelib/util.py

```diff
@@ -22,14 +22,17 @@
 import requests.auth
 import requests.exceptions
 from requests import Request
-from six import text_type, string_types
+from six import PY3, ensure_binary, string_types, text_type
 
 from .errors import TransportError, RateLimitError, RedirectError, RelativeRedirect, CASError, UnauthorizedError, \
     InvalidTokenError, ErrorInvalidSchemaVersionForMailboxVersion
 
 time_func = time.time if PY2 else time.monotonic
 log = logging.getLogger(__name__)
 
+if PY3:
+    unicode = str
```

Comment:
It looks like `unicode` is only used here for an `isinstance` check equivalent to an `ensure_binary` call. Can you make that change instead? In general, prefer `six.text_type` over redefining `unicode`.